internal/driver/config/provider.go

```diff
@@ -167,8 +167,8 @@ func (k *Config) Set(key string, v any) error {
 func (k *Config) addressFor(endpoint EndpointType) string {
 	return fmt.Sprintf(
 		"%s:%d",
-		k.p.String("serve."+string(endpoint)+".host"),
-		k.p.Int("serve."+string(endpoint)+".port"),
+		k.p.StringF("serve."+string(endpoint)+".host", ""),
+		k.p.IntF("serve."+string(endpoint)+".port", 0),
```

Comment:
Are these defaults defined in the schema, do we get them because of the zero-value, or why do we remove it here?

Reply:
Yes, the defaults from the schema are used when the config is empty. These fallbacks would be used if there were no defaults.

Note that this is a workaround for an issue in ory/x: https://github.com/ory/x/issues/664